src/biogeophys/SoilStateInitTimeConstMod.F90

```diff
@@ -793,4 +810,15 @@ end function MassFracClay
 
   !------------------------------------------------------------------------------
 
+  real(r8) function MassFracClayLeung2023( clay )
+  ! Calculate the mass fraction of clay needed for dust emission, based on clay content
+  ! Based on the base Zender_2003 version, with a slight modification for Leung_2023
+      real(r8), intent(IN) :: clay ! Fraction of lay in the soil (%)
+
+      MassFracClayLeung2023 = MassFracClay( clay )
+      MassFracClayLeung2023 = 0.1_r8 + MassFracClayLeung2023 * 0.1_r8 / 0.20_r8   ! dmleung added this line to reduce the sensitivity of dust emission flux to clay fraction in DUSTMod. 5 Jul 2024
```

Comment:
Move this into one line. Shortens the code and more efficient. In a very tiny way, but easily done.